qiskit/fundamentals/8_hello_qiskit.ipynb

```diff
@@ -0,0 +1,2250 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"../../images/qiskit_header.png\" alt=\"Note: In order for images to show up in this jupyter notebook you need to select File => Trusted Notebook\" align=\"middle\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "# Hello Qiskit\n",
+    "\n",
+    "This is a series of puzzles based on programming in Qiskit. It is based on the game [Hello Quantum](https://helloquantum.mybluemix.net/), and is intended to convey basic concepts and operations.\n",
+    "\n",
+    "If you are not already using this in the IBM Q Experience, check out the interactive version [here](https://quantum-computing.ibm.com/jupyter/tutorial/fundamentals/8_hello_quantum.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Level 1: Beginning with bits\n",
+    "\n",
+    "This is a Jupyter notebook, in which you will run some puzzles and learn about quantum computing. Don't worry if you've never used a Jupyter notebook before. It just means you'll see lots of grey boxes with code in, like the one below. These are known as cells."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Hello! I'm a code cell\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You'll need to run the cells to use this tutorial. To run a cell, do the following.\n",
+    "\n",
+    "* For laptops and desktops, click on the cell and press **Shift-Enter**.\n",
+    "\n",
+    "* For mobile devices, tap on the icon that appears to the left of a cell.\n",
```

Comment:
Do we need to point out mobile devices here?  Not sure if that works with Jupyter notebooks.